addons/clones/addon.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "Clone counter",
+  "description": "Adds a counter in the editor which shows the total amount of clones.",
+  "warning": "This feature might make Scratch projects lag, especially if you're not using a powerful computer. Performance will be enhanced in future updates.",
+  "credits": [
+    {
+      "name": "Jeffalo"
+    }
+  ],
+  "userscripts": [
+    {
+      "url": "userscript.js",
+      "matches": ["https://scratch.mit.edu/projects/*"]
+    }
+  ],
+  "userstyles": [
+    {
+      "url": "style.css",
+      "matches": ["https://scratch.mit.edu/projects/*"]
+    }
+  ],
+  "settings": [
+    {
+      "name": "Show the amount out of 300",
```

Comment:
What does this setting do?

Reply:
the maximum number of clones scratch will render is 300, so it shows the number of clones out of 300.

eg. clones: 10/300

Reply:
Is it 300 for each sprite, or in total? If in total, I'd say you remove this setting and always show "/300"

Reply:
it's for total, but it takes up a lot of space in an already small bar. i think it might actually be better to remove it and not have it as a feature

Reply:
I agree